test/index.js

```diff
@@ -545,6 +545,28 @@ describe('def', function() {
                    'as its first argument; received ["abc"]'));
   });
 
+  it('supports "enum" types', function() {
+    var def = $.create($.env.concat([$.Enum, $.ValidDate, $.ValidNumber]));
```

Comment:
I'm happy with either `TimeUnit` or `$TimeUnit`. In Sanctuary my hand was forced, as I wanted to define Maybe and Either types but the `Maybe` and `Either` identifiers were already taken. :)